SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.
+
+
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+If you find a security vulnerability we would like to here from you asap.
+Keeping Openshot and our users safe is very important to us.
+
+If you do find a security vulnerability you can report it to jonathan@openshot.org.
+It should take us a day to the latest a week to get back to you. 
+If we do comfirm it to be a real ssecurity vulnerability we will get a patch out hopefully asap. 
+
+Thank you for taking the time to care about Openshot and the saftey of it.
```

Comment:
```suggestion
Thank you for taking the time to care about OpenShot and its users.
```
("Safety" is a loaded word, it's impossible to define in these contexts. Say your email account is hacked. Does that threaten your _safety_, necessarily? Well, it **might**, or it might not, in a lot of different ways. Depends on who hacked it, what they saw, and what they intend to do with it. Etc...)